plugins/ingest-attachment/build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
   // MS Office
   api "org.apache.poi:poi-scratchpad:${versions.poi}"
   // Apple iWork
-  api 'org.apache.commons:commons-compress:1.19'
+  api 'org.apache.commons:commons-compress:1.21'
```

Comment:
As far as I can see this is the only use of this dependency in actual production code. Everything else is either build logic or test code. Does that seem right?

Reply:
Yep that's correct